src/socket.js

```diff
@@ -2,8 +2,17 @@ const socketio = require('socket.io')
 const authMiddleware = require('./../middleware/authenticationSocket')
 const { listPartnerUsers, userTypingStatus } = require('./controllers/message')
 
+const originURL =
+    process.env.NODE_ENV === 'production'
```

Comment:
I think in a previous ticket @larasodjati added IS_LOCAL to the env file. Can we use that for consistency here?